guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -262,7 +264,7 @@ The type of the object in whose context a particular method is called. Required
 );
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-type">@type</h4>
+#### @type
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-type}` to preserve the custom id.